lib/job.js

```diff
@@ -133,6 +137,20 @@ Job.prototype.toData = function() {
   const json = this.toJSON();
 
   json.data = JSON.stringify(json.data);
+
+  if (
+    this.opts.sizeLimit &&
+    utils.lengthInUtf8Bytes(json.data) > this.opts.sizeLimit
+  ) {
+    throw new Error(
+      'The size of job ' +
```

Comment:
use es6 string interpolation

Reply:
I was going to use string interpolation, but I saw several other places such as 

```javascript
throw new Error('Could not remove job ' + job.id);
```

Which didn't, so I wanted to be consistent. Should I update all of the other places as well, or do you want me to just update this one place?

Reply:
yes, that is because we just upgraded the minimum required nodejs version